### PR TITLE
create: exhibitor and talk page

### PR DESCRIPTION
**Pull request by sumaiya2908**
Add exhibitor and talk page with dynamic routes

### CODE REVIEW

File: community/www/talk/talk.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    talk = None
+
+    try:
+        talk = frappe.form_dict["talk"]
+    except KeyError:
+        frappe.local.flags.redirect_location = "/talk"
+        raise frappe.Redirect
+
+    context.talk = frappe.get_doc("Talk", talk)
+    context.talks = frappe.get_all("Talk")
```
Comment (rmehta) [inline community/www/talk/talk.py:15]: this should be filtered by conference?

**Review by rmehta — CHANGES_REQUESTED** [review]

File: community/event_management/doctype/exhibitor/exhibitor.json [hunk 2]
```diff
@@ -36,32 +37,36 @@
    "default": "0",
    "fieldname": "is_paid",
    "fieldtype": "Check",
-   "label": "Is Paid"
+   "label": "is_paid"
```
Comment (pateljannat) [inline community/event_management/doctype/exhibitor/exhibitor.json:40]: Label shouldn't be in camel case

File: community/event_management/doctype/exhibitor/exhibitor.json [hunk 3]
```diff
@@ -36,32 +37,36 @@
...
   },
   {
    "fieldname": "user",
    "fieldtype": "Link",
    "label": "User",
-   "options": "User"
+   "options": "User",
+   "read_only": 1
```
Comment (pateljannat) [inline community/event_management/doctype/exhibitor/exhibitor.json:47]: Let's not keep this read-only in the doctype. Our team might add data from the backend. The user field won't be visible to them if it's kept read-only. It should be read-only just in the web forms. Same for the Event field

File: community/event_management/doctype/speaker/speaker.json [hunk 4]
```diff
@@ -41,12 +49,45 @@
...
+   "label": "Profile Image"
+  },
+  {
+   "fieldname": "bio",
+   "fieldtype": "Text",
+   "label": "Bio"
+  },
+  {
+   "fieldname": "data_11",
+   "fieldtype": "Data"
```
Comment (pateljannat) [inline community/event_management/doctype/speaker/speaker.json:85]: Unused field?

File: community/event_management/doctype/talk/talk.json [hunk 5]
```diff
@@ -61,54 +56,29 @@
...
-   "options": "Pending\nApproved\nRejected"
-  },
-  {
-   "fieldname": "column_break_5",
-   "fieldtype": "Column Break"
-  },
-  {
-   "fieldname": "section_break_9",
-   "fieldtype": "Section Break"
+   "options": "Applied\nPending\nApproved\nRejected\nPrevious Talk"
```
Comment (pateljannat) [inline community/event_management/doctype/talk/talk.json:64]: What's the difference between Applied and Pending? Why do we need a status called Previous Talk?

**Review by pateljannat — CHANGES_REQUESTED** [review]

File: community/event_management/web_form/exhibitor_registration/exhibitor_registration.json [hunk 6]
```diff
@@ -44,7 +44,19 @@
    "max_length": 0,
    "max_value": 0,
    "options": "User",
-   "read_only": 1,
```
Comment (rmehta) [inline community/event_management/web_form/exhibitor_registration/exhibitor_registration.json:47]: not a good idea to hardcode webforms either 😅

**Pull request closed without merging**